// @coreui/angular/esm2020/lib/toast/toast.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { ButtonModule } from '../button';
import { ToastComponent } from './toast/toast.component';
import { ToastHeaderComponent } from './toast-header/toast-header.component';
import { ToastBodyComponent } from './toast-body/toast-body.component';
import { ToasterComponent } from './toaster/toaster.component';
import { ToasterService } from './toaster/toaster.service';
import { ToasterHostDirective } from './toaster/toaster-host.directive';
import { ToastCloseDirective } from './toast-close.directive';
import * as i0 from "@angular/core";
export class ToastModule {
}
ToastModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ToastModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
ToastModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ToastModule, declarations: [ToastComponent,
        ToastHeaderComponent,
        ToastBodyComponent,
        ToasterComponent,
        ToasterHostDirective,
        ToastCloseDirective], imports: [CommonModule, ButtonModule], exports: [ToastComponent,
        ToastHeaderComponent,
        ToastBodyComponent,
        ToasterComponent,
        ToasterHostDirective,
        ToastCloseDirective] });
ToastModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ToastModule, providers: [ToasterService], imports: [[CommonModule, ButtonModule]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ToastModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        ToastComponent,
                        ToastHeaderComponent,
                        ToastBodyComponent,
                        ToasterComponent,
                        ToasterHostDirective,
                        ToastCloseDirective
                    ],
                    imports: [CommonModule, ButtonModule],
                    providers: [ToasterService],
                    exports: [
                        ToastComponent,
                        ToastHeaderComponent,
                        ToastBodyComponent,
                        ToasterComponent,
                        ToasterHostDirective,
                        ToastCloseDirective
                    ]
                }]
        }] });
