// @coreui/angular/esm2020/lib/pagination/pagination.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { PaginationComponent } from './pagination/pagination.component';
import { PageItemComponent } from './page-item/page-item.component';
import { PageItemDirective } from './page-item/page-item.directive';
import { PageLinkDirective } from './page-link/page-link.directive';
import * as i0 from "@angular/core";
export class PaginationModule {
}
PaginationModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: PaginationModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
PaginationModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: PaginationModule, declarations: [PaginationComponent,
        PageItemComponent,
        PageItemDirective,
        PageLinkDirective], imports: [CommonModule], exports: [PaginationComponent,
        PageItemComponent,
        PageItemDirective,
        PageLinkDirective] });
PaginationModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: PaginationModule, imports: [[
            CommonModule
        ]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: PaginationModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        PaginationComponent,
                        PageItemComponent,
                        PageItemDirective,
                        PageLinkDirective
                    ],
                    exports: [
                        PaginationComponent,
                        PageItemComponent,
                        PageItemDirective,
                        PageLinkDirective
                    ],
                    imports: [
                        CommonModule
                    ]
                }]
        }] });
