// @coreui/angular/esm2020/lib/modal/modal.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { A11yModule } from '@angular/cdk/a11y';
import { ModalBodyComponent } from './modal-body/modal-body.component';
import { ModalContentComponent } from './modal-content/modal-content.component';
import { ModalDialogComponent } from './modal-dialog/modal-dialog.component';
import { ModalToggleDirective } from './modal-dismiss/modal-toggle.directive';
import { ModalFooterComponent } from './modal-footer/modal-footer.component';
import { ModalHeaderComponent } from './modal-header/modal-header.component';
import { ModalTitleDirective } from './modal-title/modal-title.directive';
import { ModalComponent } from './modal/modal.component';
import { ModalService } from './modal.service';
import * as i0 from "@angular/core";
export class ModalModule {
}
ModalModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ModalModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
ModalModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ModalModule, declarations: [ModalBodyComponent,
        ModalContentComponent,
        ModalDialogComponent,
        ModalToggleDirective,
        ModalFooterComponent,
        ModalHeaderComponent,
        ModalTitleDirective,
        ModalComponent], imports: [CommonModule,
        A11yModule], exports: [ModalBodyComponent,
        ModalContentComponent,
        ModalDialogComponent,
        ModalToggleDirective,
        ModalFooterComponent,
        ModalHeaderComponent,
        ModalTitleDirective,
        ModalComponent] });
ModalModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ModalModule, providers: [
        ModalService
    ], imports: [[
            CommonModule,
            A11yModule
        ]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ModalModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        ModalBodyComponent,
                        ModalContentComponent,
                        ModalDialogComponent,
                        ModalToggleDirective,
                        ModalFooterComponent,
                        ModalHeaderComponent,
                        ModalTitleDirective,
                        ModalComponent,
                    ],
                    exports: [
                        ModalBodyComponent,
                        ModalContentComponent,
                        ModalDialogComponent,
                        ModalToggleDirective,
                        ModalFooterComponent,
                        ModalHeaderComponent,
                        ModalTitleDirective,
                        ModalComponent,
                    ],
                    imports: [
                        CommonModule,
                        A11yModule
                    ],
                    providers: [
                        ModalService
                    ]
                }]
        }] });
