// @coreui/angular/esm2020/lib/grid/grid.module.mjs
import { CommonModule } from '@angular/common';
import { NgModule } from '@angular/core';
import { ContainerComponent } from './container.component';
import { RowComponent } from './row.component';
import { ColComponent } from './col.component';
import { RowDirective } from './row.directive';
import { ColDirective } from './col.directive';
import { GutterDirective } from './gutter.directive';
import * as i0 from "@angular/core";
export class GridModule {
}
GridModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: GridModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GridModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: GridModule, declarations: [ContainerComponent,
        RowComponent,
        ColComponent,
        RowDirective,
        ColDirective,
        GutterDirective], imports: [CommonModule], exports: [ContainerComponent,
        RowComponent,
        ColComponent,
        GutterDirective,
        RowDirective,
        ColDirective] });
GridModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: GridModule, providers: [], imports: [[CommonModule]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: GridModule, decorators: [{
            type: NgModule,
            args: [{
                    imports: [CommonModule],
                    exports: [
                        ContainerComponent,
                        RowComponent,
                        ColComponent,
                        GutterDirective,
                        RowDirective,
                        ColDirective,
                    ],
                    declarations: [
                        ContainerComponent,
                        RowComponent,
                        ColComponent,
                        RowDirective,
                        ColDirective,
                        GutterDirective,
                    ],
                    providers: [],
                }]
        }] });
