// @coreui/angular/esm2020/lib/button-group/button-group.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { ButtonGroupComponent } from './button-group/button-group.component';
import { ButtonToolbarComponent } from './button-toolbar/button-toolbar.component';
import * as i0 from "@angular/core";
export class ButtonGroupModule {
}
ButtonGroupModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ButtonGroupModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
ButtonGroupModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ButtonGroupModule, declarations: [ButtonGroupComponent,
        ButtonToolbarComponent], imports: [CommonModule], exports: [ButtonGroupComponent,
        ButtonToolbarComponent] });
ButtonGroupModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ButtonGroupModule, imports: [[
            CommonModule
        ]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ButtonGroupModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        ButtonGroupComponent,
                        ButtonToolbarComponent
                    ],
                    imports: [
                        CommonModule
                    ],
                    exports: [
                        ButtonGroupComponent,
                        ButtonToolbarComponent
                    ]
                }]
        }] });
