// @coreui/angular/esm2020/lib/utilities/utilities.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { BgColorDirective } from './bg-color.directive';
import { BorderDirective } from './border.directive';
import { RoundedDirective } from './rounded.directive';
import { TextColorDirective } from './text-color.directive';
import { AlignDirective } from './align.directive';
import * as i0 from "@angular/core";
export class UtilitiesModule {
}
UtilitiesModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: UtilitiesModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
UtilitiesModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: UtilitiesModule, declarations: [BgColorDirective,
        BorderDirective,
        RoundedDirective,
        TextColorDirective,
        AlignDirective], imports: [CommonModule], exports: [BgColorDirective,
        BorderDirective,
        RoundedDirective,
        TextColorDirective,
        AlignDirective] });
UtilitiesModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: UtilitiesModule, imports: [[
            CommonModule
        ]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: UtilitiesModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        BgColorDirective,
                        BorderDirective,
                        RoundedDirective,
                        TextColorDirective,
                        AlignDirective
                    ],
                    imports: [
                        CommonModule
                    ],
                    exports: [
                        BgColorDirective,
                        BorderDirective,
                        RoundedDirective,
                        TextColorDirective,
                        AlignDirective
                    ]
                }]
        }] });
