// @coreui/angular/esm2020/lib/accordion/accordion.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { AccordionComponent } from './accordion/accordion.component';
import { AccordionButtonDirective } from './accordion-button/accordion-button.directive';
import { AccordionItemComponent } from './accordion-item/accordion-item.component';
import { AccordionService } from './accordion.service';
import { CollapseModule } from '../collapse';
import { SharedModule } from '../shared';
import * as i0 from "@angular/core";
export class AccordionModule {
}
AccordionModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: AccordionModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
AccordionModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: AccordionModule, declarations: [AccordionComponent,
        AccordionButtonDirective,
        AccordionItemComponent], imports: [CommonModule,
        CollapseModule,
        SharedModule], exports: [AccordionComponent,
        AccordionButtonDirective,
        AccordionItemComponent] });
AccordionModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: AccordionModule, providers: [
        AccordionService
    ], imports: [[
            CommonModule,
            CollapseModule,
            SharedModule
        ]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: AccordionModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        AccordionComponent,
                        AccordionButtonDirective,
                        AccordionItemComponent,
                    ],
                    imports: [
                        CommonModule,
                        CollapseModule,
                        SharedModule
                    ],
                    exports: [
                        AccordionComponent,
                        AccordionButtonDirective,
                        AccordionItemComponent,
                    ],
                    providers: [
                        AccordionService
                    ]
                }]
        }] });
