// @coreui/angular/esm2020/lib/progress/progress.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { ProgressComponent } from './progress.component';
import { ProgressBarComponent } from './progress-bar.component';
import * as i0 from "@angular/core";
export class ProgressModule {
}
ProgressModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ProgressModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
ProgressModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ProgressModule, declarations: [ProgressComponent, ProgressBarComponent], imports: [CommonModule], exports: [ProgressComponent,
        ProgressBarComponent] });
ProgressModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ProgressModule, imports: [[
            CommonModule
        ]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.3", ngImport: i0, type: ProgressModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [ProgressComponent, ProgressBarComponent],
                    exports: [
                        ProgressComponent,
                        ProgressBarComponent
                    ],
                    imports: [
                        CommonModule
                    ]
                }]
        }] });
